storage/scratchpad/src/sparse_merkle/sparse_merkle_test.rs

```diff
@@ -714,6 +714,50 @@ fn test_update_consistency() {
     }
 }
 
+#[test]
+fn test_update_consistency_batches() {
```

Comment:
Could you update the prop test following (`test_read_partial_tree_from_storage()`) to call `batches_update()` can assert result is the same with `batch_update()` and `seriral_update()` as well?